go/master/client.go

```diff
@@ -60,13 +60,31 @@ func WithAddr(addr string) func(c *Client) error {
 	}
 }
 
+func retry(f func() error, dur time.Duration, count int) error {
+	err := f()
+	if err != nil {
+		if count > 0 {
+			time.Sleep(dur)
+			return retry(f, dur, count-1)
```

Comment:
I see, make sense, will change to wait forever.

I got this error when trying Paddle cloud with fault tolerant on, that is why I am trying to change here:

```
➜  paddle git:(develop) ✗ paddlecloud logs -n 1000 b                                                                                           
==========================b-trainer-brwn0==========================
time="2017-08-23T23:42:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:42:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:42:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:42:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:42:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:42:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:42:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:43:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:44:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:45:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:46:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:47:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:48:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:49:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:50:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:51:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:52:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:53:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:54:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:55:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:56:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:57:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:04Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:09Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:14Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:19Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:24Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:29Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:34Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:39Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:44Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:49Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:54Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:58:59Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-23T23:59:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:00:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:01:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:02:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:03:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:04:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:05:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:06:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:07:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:08:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:09:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:10:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:11:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:12:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:13:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:14:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:15:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:16:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:17:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:18:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:19:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:20:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:21:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:05Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:10Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:15Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:20Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:25Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:30Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:35Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:40Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:45Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:50Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:22:55Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:00Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:23:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:24:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:25:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:26:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:27:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:28:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:29:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:30:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:31:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:32:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:33:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:34:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:35:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:36:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:37:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:38:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:39:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:40:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:41:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:42:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:43:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:44:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:45:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:06Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:11Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:16Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:21Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:26Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:31Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:36Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:41Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:46Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:51Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:46:56Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:01Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:47:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:48:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:49:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:50:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:51:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:52:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:53:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:54:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:55:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:56:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:57:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:58:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T00:59:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:00:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:01:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:02:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:03:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:47Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:52Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:04:57Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:05:02Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:05:07Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:05:12Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:05:17Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:05:22Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:05:27Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:05:32Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:05:37Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:05:42Z" level=info msg="Waiting for ps desired registered ..." 
time="2017-08-24T01:05:47Z" level=info msg="Waiting for ps desired registered ..." 
==========================b-trainer-lbs8m==========================
label selector: paddle-job-master=b, desired: 1
label selector: paddle-job=b, desired: 1
Starting training job:  /pfs/dlnel/home/helinwang@baidu.com/jobs/b, num_gradient_servers: 1, trainer_id:  0, version: 
I0823 20:36:07.662369    34 Util.cpp:166] commandline:  --num_gradient_servers=1 --ports_num_for_sparse=1 --use_gpu=0 --trainer_id=0 --trainer_count=1 --num_passes=1 --ports_num=1 --port=7164 
I0823 20:36:07.668710    34 GradientMachine.cpp:85] Initing parameters..
I0823 20:36:07.668742    34 GradientMachine.cpp:92] Init parameters done.
panic: dial tcp 10.1.93.6:2379: getsockopt: connection refused
goroutine 17 [running, locked to thread]:
main.paddle_new_etcd_master_client(0x7faf728eebd4, 0x5, 0x40, 0x1c400000008)
	/paddle/build/go/src/github.com/PaddlePaddle/Paddle/go/master/c/client.go:79 +0x132
main._cgoexpwrap_f2aa1382a54e_paddle_new_etcd_master_client(0x7faf728eebd4, 0x5, 0x40, 0xd74167732b8bc700)
	command-line-arguments/_obj/_cgo_gotypes.go:100 +0x41
Aborted (core dumped)
job returned 134...setting pod return message...
===============================
termination log wroted...
```